server/src/main/java/org/elasticsearch/snapshots/SnapshotsService.java

```diff
@@ -132,6 +133,26 @@
 
     public static final String UPDATE_SNAPSHOT_STATUS_ACTION_NAME = "internal:cluster/snapshot/update_snapshot_status";
 
+    public static final String SEARCHABLE_SNAPSHOT_SETTINGS_PREFIX = "xpack.searchable.snapshot.shared_cache.";
```

Comment:
this is the prefix for the shared cache, not for all SEARCHABLE_SNAPSHOT_SETTINGS

Reply:
++ renamed.